notes/mfip-orientation.vbs

```diff
@@ -0,0 +1,957 @@
+						CAREGIVER_ARRAY(orientation_needed_const, person) = False
+						CAREGIVER_ARRAY(orientation_exempt_const, person) = True
+						CAREGIVER_ARRAY(exemption_reason_const, person) = "2nd Caregiver Employed 20+ hours per week"
+						CAREGIVER_ARRAY(emps_exemption_code_const, person) = "21"
+					End If
+				End If
+				If CAREGIVER_ARRAY(memb_age_const, person) < 20 Then
+					minor_caregiver_on_case = minor_caregiver_on_case + 1
+					CAREGIVER_ARRAY(exempt_from_ed_const, person) = "No"
+					CAREGIVER_ARRAY(comply_with_ed_const, person) = "Yes"
+				End If
+
+			End If
+
+			If CAREGIVER_ARRAY(memb_name_const, person) = caregiver_two Then
+				CAREGIVER_ARRAY(memb_is_caregiver, person) = True
+				CAREGIVER_ARRAY(orientation_needed_const, person) = True
+
+				If caregiver_two_req_cash = "Yes" Then CAREGIVER_ARRAY(cash_request_const, person) = True
+				If caregiver_two_req_cash <> "Yes" Then
+					CAREGIVER_ARRAY(cash_request_const, person) = False
+					CAREGIVER_ARRAY(orientation_needed_const, person) = False
+					CAREGIVER_ARRAY(orientation_exempt_const, person) = True
+					CAREGIVER_ARRAY(exemption_reason_const, person) = "Caregiver Not on MFIP"
+					CAREGIVER_ARRAY(emps_exemption_code_const, person) = "NO"
+				End If
+				CAREGIVER_ARRAY(hours_per_week_const, person) = caregiver_two_hours_per_week
+
+				If CAREGIVER_ARRAY(hours_per_week_const, person) > 34 Then
+					CAREGIVER_ARRAY(orientation_needed_const, person) = False
+					CAREGIVER_ARRAY(orientation_exempt_const, person) = True
+					CAREGIVER_ARRAY(exemption_reason_const, person) = "Employed 35+ hours per week"
+					CAREGIVER_ARRAY(emps_exemption_code_const, person) = "20"
+				ElseIf CAREGIVER_ARRAY(hours_per_week_const, person) > 19 Then
+					If caregiver_one <> "No Second Caregiver" AND caregiver_one_req_cash = "Yes" AND caregiver_one_hours_per_week > 34 Then
+						CAREGIVER_ARRAY(orientation_needed_const, person) = False
+						CAREGIVER_ARRAY(orientation_exempt_const, person) = True
+						CAREGIVER_ARRAY(exemption_reason_const, person) = "2nd Caregiver Employed 20+ hours per week"
+						CAREGIVER_ARRAY(emps_exemption_code_const, person) = "21"
+					End If
+				End If
+				If CAREGIVER_ARRAY(memb_age_const, person) < 20 Then
+					minor_caregiver_on_case = minor_caregiver_on_case + 1
+					CAREGIVER_ARRAY(exempt_from_ed_const, person) = "No"
+					CAREGIVER_ARRAY(comply_with_ed_const, person) = "Yes"
+				End If
+			End If
+
+
+
+		Next
+
+		'IF A MINOR IS FOUND
+		If minor_caregiver_on_case > 0 Then
+			Do
+				err_msg = ""
+				dlg_len = 210
+				If minor_caregiver_on_case = 2 Then dlg_len = 290
+
+				Dialog1 = ""
+				BeginDialog Dialog1, 0, 0, 551, dlg_len, "Assess for Caregiver MFIP Orientation Requirement"
+				  Text 10, 15, 200, 10, "Which Family Cash Program is this Application for? " & family_cash_program
+				  Text 10, 25, 500, 20, "Notes on Program Selection: " & famliy_cash_notes
```

Comment:
family_cash_notes: This doesn't export to case note.

Reply:
UGH - yes - this became a whole thing - should be done now.